app/elements/gde-signin/gde-signin.html

```diff
@@ -56,37 +64,95 @@
           /**
            * User information from Google authentication
            */
-          _googleUser: {
-            type: Object,
-            computed: '_getBasicProfile(_isAuthorized)'
-          },
+          _googleUser: Object,
 
           /**
            * User information from the GDE Tracking App backend
            */
           _apiUser: Object,
+
+          /**
+           * User information from local storage
+           */
+          _localUser: Object,
+
+          /**
+           * User has granted Google authorization
+           */
+          _isAuthorized: {
+            type: Boolean,
+            value: false
+          }
         },
 
         /**
-         * Retrieve user information from Google authentication
+         * User signed in, retrieve user information from Google authentication
          */
-        _getBasicProfile: function (isAuthorized) {
-          if (!isAuthorized) { return undefined; }
+        _signedIn: function () {
           var profile = gapi.auth2.getAuthInstance()
                           .currentUser.get().getBasicProfile();
-          return {
+
+          var authResponse = gapi.auth2.getAuthInstance()
+                            .currentUser.get().getAuthResponse();
+
+          this._googleUser = {
             id: profile.getId(),
             name: profile.getName(),
             image: profile.getImageUrl(),
-            email: profile.getEmail()
+            email: profile.getEmail(),
+            token: authResponse.access_token || authResponse.id_token
           };
+          this._isAuthorized = true;
+          this._localUser = {
+            id: profile.getId(),
+            name: profile.getName(),
+            image: profile.getImageUrl(),
+            email: profile.getEmail(),
+            token: authResponse.access_token || authResponse.id_token
+          };
+        },
+
+        /**
+         * User signed out, remove all user information
+         */
+        _signedOut: function () {
+          this._googleUser = undefined;
+          this._apiUser = undefined;
+          this._localUser = {};
+          this._isAuthorized = false;
+          this._setUser({});
```

Comment:
I think there is and issue regarding undefined... but obviously I can't find the issue...
Instead I've found [this](https://github.com/Polymer/polymer/issues/1813) TL;DR; we should try out if setting the objects to null instead of undefined triggers the computation correctly and the conditions still works correctly